config_connector/README.md

```diff
@@ -0,0 +1,9 @@
+# Cloud Connector
+
+This directory contains configurations and scripts for using [Config Connector](https://cloud.google.com/config-connector/) to setup Kubernetes clusters.
+
+## Setup
+
+Run `config_connector_setup.sh` to setup Config Connector on a GKE cluster.
+
```

Comment:
Can you add more information about how this is setup?

e.g. what the folder and project organization is? What information would a person need to know in order to fix any problems and continue to iterate on this?